src/editor/user-tasks/task-definitions.js

```diff
@@ -120,16 +110,4 @@ tasks.toggleHelpModal = new UserTask({
   }
 });
 
-tasks.fileAnIssue = new ExternalLinkTask({
```

Comment:
i'm pretty sure these two tasks still need to be defined?

Reply:
They aren't referenced anywhere in the UI, I think we're best off removing them.

Reply:
https://github.com/iodide-project/iodide/blob/b846ff15c5b5fe166e5adc3377bf8308357838f2/src/editor/components/menu/notebook-task-button.jsx#L6

https://github.com/iodide-project/iodide/blob/b846ff15c5b5fe166e5adc3377bf8308357838f2/src/editor/components/menu/notebook-menu-item.jsx#L6

Reply:
Those aren't examples of usage of those tasks? That's just importing the class from which they are derived.

Reply:
you're totally right, my mistake